lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -653,20 +596,51 @@ def test_coefficient_values_for_gaussian_distribution(self):
         """Ensure that the values for the optimised_coefficients match the
         expected values, and the coefficient names also match
         expected values for a Gaussian distribution."""
-        data = [4.55819380e-06, -8.02401974e-09,
-                1.66667055e+00, 1.00000011e+00]
+        plugin = Plugin(self.distribution, self.current_cycle)
+        result = plugin.estimate_coefficients_for_ngr(
+            self.historic_temperature_forecast_cube,
+            self.temperature_truth_cube)
+
+        self.assertArrayAlmostEqualLowerPrecision(
+            result.data, self.expected_mean_predictor_gaussian)
+        self.assertArrayEqual(
+            result.coord("coefficient_name").points, self.coeff_names)
+
+    @ManageWarnings(
+        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
+    def test_coefficients_gaussian_distribution_default_initial_guess(self):
+        """Ensure that the values for the optimised_coefficients match the
+        expected values, and the coefficient names also match
+        expected values for a Gaussian distribution."""
+        expected = [-0.00023604, 0.79765, 0.00042286, 0.99733]
```

Comment:
Why this test is different from the test above?
- This test is different because it uses the default initial guess, rather than the initial guess computed using linear regression. I've updated the docstring the clarify this.
Why the expected values are significantly different from the test above?
- The differences are related to the different initial guesses. For comparison, the initial guess when using linear regression is [0., 1., 25.7496, 0.9047], whilst the default initial guess is [0, 1, 0, 1].
Please change the expected values to correspond to the level of precision at which you are testing them.
- This has been corrected.